SimulationAgent/appsettings.ini

```diff
@@ -5,12 +5,37 @@ iothub_connstring = "${PCS_IOTHUB_CONNSTRING}"
 device_models_folder = ./data/devicemodels/
 device_models_scripts_folder = ./data/devicemodels/scripts/
 
+[DeviceSimulationService:RateLimits]
+# S3:  5000/min/unit (= 83.3/sec/unit)
+# S2:   100/min/unit (= 1.67/sec/unit)
+# S1:   100/min/unit (= 1.67/sec/unit)
+# F1:   100/min      (= 1.67/sec)
+registry_operations_per_minute = 100
+# S3:  50/sec/unit
+# S2:   1/sec/unit  (10/sec minimum)
+# S1:  10/sec
+# F1:  10/sec
+twin_reads_per_second = 10
+# S3:  50/sec/unit
+# S2:   1/sec/unit  (10/sec minimum)
+# S1:  10/sec
+# F1:  10/sec
+twin_writes_per_second = 10
+# S3:  6000/sec/unit
+# S2:   120/sec/unit
+# S1:    12/sec/unit (100/sec minimum)
+# F1:   100/sec
+device_connections_per_second = 100
+# S3:  300M/day/unit (= 208333.33/min/day)
+# S2:    6M/day/unit (=   4166.67/min/unit)
+# S1:  400k/day/unit (=    277.78/min/unit)
+# F1:  8000/day      (=      5.56/min)
+messages_per_day = 400000
```

Comment:
300k?  we should leave room for physical devices.

Reply:
see https://github.com/Azure/device-simulation-dotnet/issues/93